server/systemroutes.js

```diff
@@ -79,9 +80,7 @@ router.get('/_monitor', _monitor)
 router.get('/_monitor_all', _monitorAll)
 router.get('/_monitor_core', function (req, res) {
   res.setHeader('Content-Type', 'text/plain')
-  res.send(stripIndent`
-    APPLICATION_STATUS: OK
-  `)
+  res.send('APPLICATION_STATUS: OK')
```

Comment:
Why was stripIndent used here before? Did we want a leading and ending "newline"?

Reply:
I think there _should_ be a trailing newline, but not a leading.  Maybe just put a `\n` at the end of the string?

Reply:
@kaj : Sounds correct. Let's try it!